spec/unit/pdk/util/template_uri_spec.rb

```diff
@@ -302,37 +302,45 @@
   end
 
   describe '.default_template_ref' do
-    subject { described_class.default_template_ref }
+    subject { described_class.default_template_ref(uri) }
```

Comment:
I don't see any tests for `described_class.default_template_ref()` anywhere.  Should these be added?

Reply:
I can add an explicit `nil` test if you want, but it's just going to be duplicating the tests here https://github.com/puppetlabs/pdk/blob/e8797f535cb55b2fc29b195f80e54db7f5bcf8eb/spec/unit/pdk/util/template_uri_spec.rb#L331-L349

Reply:
The only thing I would be looking for is:
`expect(described_class.default_template_ref()).to eq(PDK::TEMPLATE_REF)`